manifests/init.pp

```diff
@@ -430,6 +439,9 @@
   String                                          $default_logging_level     = $logging_level,
   Optional[String]                                $keystore_password         = undef,
   Optional[Stdlib::Absolutepath]                  $keystore_path             = undef,
+  Optional[String]                                $logging_content           = undef,
```

Comment:
May be change this to `Optional[Variant[String, Array]]` to support yaml sequence in hiera data?